docs: add yozefu (#868)
ratatui-org/ratatui-website: src/content/docs/showcase/apps.md

documentations in your terminal.

![openapi-tui demo](https://github.com/zaghaghi/openapi-tui/blob/main/static/demo.gif?raw=true)

---

## [`yozefu`](https://github.com/MAIF/yozefu)

> Yōzefu is an interactive terminal user interface (TUI) application for exploring data of a kafka cluster. It is an alternative tool to AKHQ, redpanda console or the kafka plugin for JetBrains IDEs. It includes a search query language inspired from SQL providing a fine-grained way filtering capabilities.

Reviewer: ```suggestion
Yōzefu is an interactive TUI application for exploring data of a Kafka cluster.

It is an alternative tool to AKHQ, redpanda console or the kafka plugin for JetBrains IDEs. It includes a search query language inspired from SQL providing a fine-grained way filtering capabilities.
```

documentations in your terminal.

![openapi-tui demo](https://github.com/zaghaghi/openapi-tui/blob/main/static/demo.gif?raw=true)

---

## [`yozefu`](https://github.com/MAIF/yozefu)

Yōzefu is an interactive TUI application for exploring data of a Kafka cluster.

> It is an alternative tool to AKHQ, redpanda console or the kafka plugin for JetBrains IDEs. It includes a search query language inspired from SQL providing a fine-grained way filtering capabilities.

Reviewer 2: ```suggestion
It is an alternative tool to AKHQ, Redpanda Console, or the Kafka plugin for JetBrains IDEs. It
includes a search query language inspired by SQL, providing fine-grained filtering capabilities.
```

> ![yozefu demo](https://vhs.charm.sh/vhs-UpIJD2h92vKkj01XSS0r0.gif)

Reviewer 2: nit: can you include this in your repo and point to that instead?

Author: I applied the changes, let me know if it's ok

Reviewer 3: I'm actually fine with vhs links - letting someone else host the gif is fine with me, as it means that the repo doesn't get bloated with image updates over time. The alternative is to upload the image to a comment in a PR or similar and then use github's hosting.

Reviewer 2: Ok, resolving then
